tox.ini

```diff
@@ -51,6 +51,15 @@ basepython = python2.6
 deps = {[testenv]basedeps}
        django>=1.5,<1.6
 
+[testenv:gae]
```

Comment:
I'll just handle it when updating `fetch_gae_sdk`. This may make people who run `tox` with no eviron angry, but that might be a total of 1 (me) right now and I'm not angry.